### ISSUE TITLE
CopyToOutput deleted when adding a new file

### ISSUE THREAD

**Issue by jchannon** [ionide/ionide-vscode-fsharp#1383]
**Describe the bug**
If I have a fsproj like so:
```
  <ItemGroup>
    <Content Include="customers.json" CopyToOutputDirectory="Always" />
    <Compile Include="DataStore.fs" />
    <Compile Include="RestrictionOperators.fs" />
    <Compile Include="ProjectionOperators.fs" />
    <Compile Include="PartitioningOperators.fs" />
    <Compile Include="OrderingOperators.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>
```

If I choose Add File Above, and add a new file I get:

```
  <ItemGroup>
    <Content Include="customers.json" />
    <Compile Include="DataStore.fs" />
    <Compile Include="RestrictionOperators.fs" />
    <Compile Include="ProjectionOperators.fs" />
    <Compile Include="PartitioningOperators.fs" />
    <Compile Include="OrderingOperators.fs" />
    <Compile Include="NewFile.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>
```

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Solution Explorer
2. Click on a file
3. Click add new file above
4. View fsproj

**Expected behaviour**
For it to add an entry of the new file and not to delete other attributes

**Screenshots**

**Environment (please complete the following information):**
 - OS: Mac OS
 - Ionide version: 4.12.0
 - VSCode version: 1.46.0
 - dotnet SDK version: 3.100.0
 - mono / .Net Framework version: N/A

**Comment by pihai** [comment]
There are more things which get deleted. The following screenshot shows what gets deleted from my function app fsproj:

![fsproj-delete](https://user-images.githubusercontent.com/1460563/85282400-f3aee280-b48b-11ea-86a1-8187b9492129.png)

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed with 5.0
